IntegrationTests/Configuration.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  Configuration.swift
+//  BoxSDKIntegrationTests-iOS
+//
+//  Created by Artur Jankowski on 01/12/2021.
+//  Copyright © 2021 box. All rights reserved.
+//
+
+import Foundation
+
+enum Configuration {
+    static var developerToken: String = "" // Please put your DEVELOPER TOKEN here
```

Comment:
So the developer token must be placed here every time the tests are ran? Dev tokens are valid only for 60 minutes. Are we considering support for obtaining access tokens based on client credentials?

Reply:
Yes currently is should be placed before every tests run. 
But this solution will be evolving w to OAuth access_token by using secret and client_id.